postprocessing/janus-pp-rec.c

```diff
@@ -257,9 +265,10 @@ int main(int argc, char *argv[])
 			cmdline_parser_free(&args_info);
 			exit(1);
 		}
-		if(strcasecmp(extension, ".opus") && strcasecmp(extension, ".wav") &&
-				strcasecmp(extension, ".webm") && strcasecmp(extension, ".mp4") &&
-				strcasecmp(extension, ".srt")) {
+		extension ++;
```

Comment:
Please remove the extra space there (code style).

Reply:
done